terraform.json.example

```diff
@@ -21,6 +20,23 @@
   ],
   "gpuNvidiaDriverVersion": "535.54.03",
   "infrastructurename": "simphera",
+  "ingress_nginx_config": {
+    "chart_values": {
+      "controller": {
+        "images": {
+          "registry": "registry.k8s.io"
+        },
+        "service": {
+          "annotations": {
+            "service.beta.kubernetes.io/aws-load-balancer-scheme": "internet-facing"
+          }
+        }
+      }
+    },
+    "enable": false,
```

Comment:
I would prefer to use true as default since it's more common that we need this addon then instances where we need it. Not sure what other think about it.